sale_order_mass_confirmation/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 - Today Coop IT Easy SCRLfs (<http://www.coopiteasy.be>)
+# - Grégoire Leeuwerck <gregoire@coopiteasy.be>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Mass confirmation of sale orders",
+    "summary": """
+        Confirm multiple sale orders (quotations) with one action""",
+    "description": """
+        Adds an action to confirm selected sale orders (quotations).
+    """,
+    "author": "Coop IT Easy SCRLfs",
+    "license": "AGPL-3",
+    "website": "https://coopiteasy.be/",
+    "category": "Uncategorized",
+    "version": "0.1",
```

Comment:
Use `12.0.0.1.0` as version number. `12.0` as odoo version and then a three digit version for your module. :)